src/Lecture_8/Homework_8/Lector.java

```diff
@@ -0,0 +1,20 @@
+package Lecture_8.Homework_8;
+
+public class Lector extends Person {
+    private double dailySalary;
+    public Lector(String firstName, String lastName, int age, String gender, double dailySalary){
+        super(firstName, lastName, age, gender);
+        this.dailySalary = dailySalary;
+    }
+    public String showLectorInfo() {
+        return super.showPersonInfo() + "\n" +
+                "Daily salary: " + this.dailySalary;
+    }
+    public double calculateOvertime(double hours) {
+        double overtimeAmout = 0;
+        if (getAge() >= 30) {
```

Comment:
number 30 could be a constant field and will bring more information that just using the number itself:
EXPERIENCED_LECTOR_MIN_AGE

Reply:
Got it, thank you. Added experiencedLectorMinAge as a constant.